mayastor/src/bin/mayastor-client/controller_cli.rs

```diff
@@ -48,6 +52,61 @@ fn controller_state_to_str(idx: i32) -> String {
     .to_string()
 }
 
+async fn controller_stats(
+    mut ctx: Context,
+    _matches: &ArgMatches<'_>,
+) -> crate::Result<()> {
+    let response = ctx
+        .client
+        .stat_nvme_controllers(rpc::Null {})
+        .await
+        .context(GrpcStatus)?;
+
+    match ctx.output {
+        OutputFormat::Json => {
+            println!(
+                "{}",
+                serde_json::to_string_pretty(&response.get_ref())
+                    .unwrap()
+                    .to_colored_json_auto()
+                    .unwrap()
+            );
+        }
+        OutputFormat::Default => {
+            let controllers = &response.get_ref().controllers;
+            if controllers.is_empty() {
+                ctx.v1("No NVMe controllers found");
+                return Ok(());
+            }
+
+            let table: Vec<Vec<String>> = controllers
+                .iter()
+                .map(|c| {
+                    let stats = c.stats.as_ref().unwrap();
+
+                    let num_read_ops = stats.num_read_ops.to_string();
+                    let num_write_ops = stats.num_write_ops.to_string();
+                    let bytes_read = stats.bytes_read.to_string();
+                    let bytes_written = stats.bytes_written.to_string();
+
+                    vec![
+                        c.name.to_string(),
+                        num_read_ops,
+                        num_write_ops,
+                        bytes_read,
+                        bytes_written,
+                    ]
+                })
+                .collect();
+
+            let hdr = vec!["NAME", "READS", "WRITES", "READ", "WRITTEN"];
```

Comment:
Please append '/B' to read and written headings so it's clear what units they're in.

Reply:
Good point, thanks.